src/Integration.Vsix/SonarLintDaemon/SonarLintDaemon.cs

```diff
@@ -131,23 +130,24 @@ public void Start()
             process.OutputDataReceived += (sender, args) =>
             {
                 string data = args.Data;
-                if (data != null)
+                if (data != null && data.Contains("Server started"))
                 {
-                    if (data.Contains("Server started"))
+                    if (IsVerbose())
                     {
-                        try
+                        WritelnToPane(data);
+                    }
+
+                    bool succeeded = false;
+                    SafeOperation(() =>
                         {
                             CreateChannelAndStreamLogs();
                             Ready?.Invoke(this, EventArgs.Empty);
-                        }
-                        catch (Exception ex) when (!ErrorHandler.IsCriticalException(ex))
-                        {
-                            logger.WriteLine($"Daemon error: {ex.ToString()}");
-                        }
-                    }
-                    if (IsVerbose())
```

Comment:
I moved this logging statement before the error handling block so that the logged output appears in time-order.